dialyzer-ignore.exs

```diff
@@ -0,0 +1,3 @@
+[
+  {"lib/slipstream.ex"}
```

Comment:
How is this needed? This seems like a stranger than normal dialyzer workaround.

Reply:
https://github.com/NFIBrokerage/slipstream/pull/29